src/CreateConnectorConfig.ts

```diff
@@ -96,4 +96,10 @@ function validateConnectorConfig(connectorConfig: ConnectorRuntimeConfig): void
         }
         throw new Error(errorMessage);
     }
+
+    if (!connectorConfig.transportLibrary.baseUrl.startsWith("http://") || !connectorConfig.transportLibrary.baseUrl.startsWith("https://")) {
```

Comment:
What error was shown and what error is shown now if `transportLibrary.baseUrl` is not set at all? (If `transportLibrary.baseUrl` is not set, I would like to see a different error.)

Reply:
the validator kicks in then